Content.Server/WhiteDream/BloodCult/ConstructActionsSystem.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Content.Server.WhiteDream.BloodCult.Constructs.PhaseShift;
+using Content.Shared.StatusEffect;
+using Content.Shared.WhiteDream.BloodCult.Spells;
+using Robust.Server.Audio;
+using Robust.Server.GameObjects;
+using Robust.Shared.Map;
+using Robust.Shared.Map.Components;
+using PhaseShiftedComponent = Content.Shared.WhiteDream.BloodCult.Constructs.PhaseShift.PhaseShiftedComponent;
+
+namespace Content.Server.WhiteDream.BloodCult;
+
+public sealed class ConstructActionsSystem : EntitySystem
+{
+    [Dependency] private readonly ITileDefinitionManager _tileDef = default!;
+
+    [Dependency] private readonly AudioSystem _audio = default!;
```

Comment:
you can remove the whitespace yeah?

Reply:
My religion tells my otherwise. The whitespace here is to logically separate `Managers` from `Systems`

Reply:
I respect your religion then